dashboard/test/models/section_test.rb

```diff
@@ -1,69 +1,73 @@
 require 'test_helper'
 
 class SectionTest < ActiveSupport::TestCase
-  test "do not attempt to create sections with duplicate random codes" do
-    teacher = create(:teacher)
+  setup do
+    @student = create :student
+    @teacher = create :teacher
+    @section = create :section, teacher: @teacher
 
-    srand 1
-    s1 = Section.create!(user: teacher, name: "section 1")
+    @default_attrs = {user: @teacher, name: 'test-section'}
+  end
```

Comment:
Done. Unfortunately, this forces us to use a "local" `section` for one of the tests.
```
  test_section_type_validation                                   ERROR (0.00s)
RuntimeError:         RuntimeError: can't modify frozen Hash
            test/models/section_test.rb:148:in `block in <class:SectionTest>'
            test/testing/setup_all_and_teardown_all.rb:22:in `run'
```